sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/_client.py

```diff
@@ -272,12 +272,16 @@ def get_deleted_certificate(self, certificate_name, **kwargs):
     @distributed_trace
     def purge_deleted_certificate(self, certificate_name, **kwargs):
         # type: (str, **Any) -> None
-        """Permanently deletes the specified deleted certificate.
+        """Permanently delete a certificate. Possible only in vaults with soft-delete enabled.
```

Comment:
Suggest keeping the "deleted" qualification. Phrased this way it suggests purge provides one-step permanent deletion and that permanent deletion is only possible in a soft-delete enabled vault.

Reply:
Makes sense, I'll add this to keys and secrets too